src/components/UserLabel/types.ts

```diff
@@ -0,0 +1,23 @@
+import type React from 'react';
+
+import type {DistributiveOmit} from '../../types/utils';
+import type {AvatarProps} from '../Avatar';
+import type {DOMProps, QAProps} from '../types';
+
+export type UserLabelType = 'person' | 'email' | 'empty';
+export type UserLabelText = {value: string; content: React.ReactNode} | string;
```

Comment:
Can it be just `string | React.ReactNode`?

Reply:
No, value is required because it is used for passing parameters to `onClick` and `onClose` callbacks.